docs/data_structures/libE_specs.rst

```diff
@@ -256,10 +256,10 @@ libEnsemble is primarily customized by setting options within a ``LibeSpecs`` cl
                     many GPUs.
 
                 **gpus_per_group** [int]:
-                    Number of GPUs for each group in the scheduler. This can be used to deal
-                    with scenarios where nodes have different numbers of GPUs. In effect a
+                    Number of GPUs for each group in the scheduler. This can be used when
+                    running on nodes with different numbers of GPUs. In effect a
                     block of this many GPUs will be treated as a virtual node.
-                    By default the GPUs on a node are treated as a group.
+                    By default the GPUs on each node are treated as a group.
```

Comment:
@shuds13 The last line might be confusing. Is that saying that even though the nodes might have different numbers of GPUs, the groups will be a different size by default?